eno-ws/src/main/java/fr/insee/eno/legacy/exception/EnoGenerationException.java

```diff
@@ -0,0 +1,4 @@
+package fr.insee.eno.legacy.exception;
+
+public class EnoGenerationException extends Throwable {
```

Comment:
Throwable is the super class of Error and Exception. As you are using an exception, could be better to extends Exception here

Reply:
Didn't see it when copied the classes from legacy, good point